ports/rp2/mphalport.c

```diff
@@ -83,11 +83,12 @@ int mp_hal_stdin_rx_chr(void) {
         #if MICROPY_HW_USB_CDC
         mp_usbd_cdc_poll_interfaces(0);
         #endif
-
+        #if MICROPY_HW_ENABLE_UART_REPL || MICROPY_PY_OS_DUPTERM_NOTIFY
```

Comment:
I'm not sure why this guard is added, because `shared/tinyusb/mp_usbd_cdc.c` pushes to `stdin_ringbuf`.

Reply:
But if CDC is enabled, and UART_REPL and DUPTERM_NOTIFY disabled, won't that mean input from CDC will no longer work?